pkg/kn/commands/service/migrate.go

```diff
@@ -152,11 +152,11 @@ func NewServiceMigrateCommand(p *commands.KnParams) *cobra.Command {
 
 				if serviceExists {
 					if !migrateFlags.ForceReplace {
-						fmt.Println("\n[Error] Cannot kn-migration service", color.CyanString(service_s.Name), "in namespace", color.BlueString(namespaceS),
+						fmt.Println("\n[Error] Cannot kn-migration service", service_s.Name, "in namespace", namespaceS,
```

Comment:
Change to “... Cannot migrate service ...”

Reply:
Fixed in https://github.com/knative/client/pull/406/commits/bb3fd28369fc262e8757bce812b525effe74fe3d, thanks!